tests/fsharpqa/Source/CompilerOptions/fsc/help/help40.437.1033.bsl

```diff
@@ -1,110 +1,112 @@
+                                         assembly
+--delaysign[+|-]                         Delay-sign the assembly using only the public
+                                         portion of the strong name key
+--publicsign[+|-]                        Public-sign the assembly using only the public
+                                         portion of the strong name key, and mark the
+                                         assembly as signed
+--doc:<file>                             Write the xmldoc of the assembly to the given
+                                         file
+--keyfile:<file>                         Specify a strong name key file
+--keycontainer:<string>                  Specify a strong name key container
+--platform:<string>                      Limit which platforms this code can run on: x86,
+                                         Itanium, x64, anycpu32bitpreferred, or anycpu.
+                                         The default is anycpu.
+--nooptimizationdata                     Only include optimization information essential
+                                         for implementing inlined constructs. Inhibits
+                                         cross-module inlining but improves binary
+                                         compatibility.
+--nointerfacedata                        Don't add a resource to the generated assembly
+                                         containing F#-specific metadata
+--sig:<file>                             Print the inferred interface of the assembly to
+                                         a file
+--nocopyfsharpcore                       Don't copy FSharp.Core.dll along the produced
+                                         binaries
 
 
 		- INPUT FILES -
---reference:<file>             Reference an assembly (Short form: -r)
+--reference:<file>                       Reference an assembly (Short form: -r)
 
 
 		- RESOURCES -
---win32res:<file>              Specify a Win32 resource file (.res)
---win32manifest:<file>         Specify a Win32 manifest file
---nowin32manifest              Do not include the default Win32 manifest
---resource:<resinfo>           Embed the specified managed resource
---linkresource:<resinfo>       Link the specified resource to this assembly
-                               where the resinfo format is <file>[,<string
-                               name>[,public|private]]
+--win32res:<file>                        Specify a Win32 resource file (.res)
+--win32manifest:<file>                   Specify a Win32 manifest file
+--nowin32manifest                        Do not include the default Win32 manifest
+--resource:<resinfo>                     Embed the specified managed resource
+--linkresource:<resinfo>                 Link the specified resource to this assembly
+                                         where the resinfo format is <file>[,<string
+                                         name>[,public|private]]
```

Comment:
Does the help text still look OK visually when printed on an 80-column console window?  thanks